src/CMake/FindANARI.cmake

```diff
@@ -0,0 +1,133 @@
+# Copyright (c) Lawrence Livermore National Security, LLC and other VisIt
+# Project developers.  See the top-level LICENSE file for dates and other
+# details.  No copyright assignment is required to contribute to VisIt.
+
+#****************************************************************************
+# Modifications:
+#
+#*****************************************************************************
+
+#[=======================================================================[.rst:
+FindANARI
+---------
+
+Finds the ANARI library.
+
+Imported Targets
+^^^^^^^^^^^^^^^^
+
+This module provides the following imported targets, if found:
+
+``anari::anari``
+  The ANARI library
+``anari::anari_utilities``
+  The ANARI utilities library
+``anari::anari_library_debug``
+  Library target for the debug device
+``anari::helium``
+  Library target containing base device implementation abstractions
+
+Result Variables
+^^^^^^^^^^^^^^^^
+
+This will define the following variables:
+
+``anari_FOUND``
+  True if the system has the ANARI library.
+``HAVE_ANARI_EXAMPLE``
+  True if the ANARI example back-end library (helide) was found.
+
+Cache Variables
+^^^^^^^^^^^^^^^
+
+The following cache variables may also be set:
+
+``anari_DIR``
+  The directory containing the ANARI config files.
+``ANARI_INCLUDE_DIRS``
+  The directory containing ``anari/anari.h``.
+``ANARI_LIBRARIES``
+  The ANARI Libraries and third-party ANARI back-end libraries if found
+``ANARI_Example_LIBRARY``
+  The path for the ANARI Example back-end library
+
+#]=======================================================================]
+
+if(EXISTS ${VISIT_ANARI_DIR})
+    message(STATUS "Checking for ANARI in ${VISIT_ANARI_DIR}/lib/cmake/anari-${ANARI_VERSION}")
+
+    if(NOT DEFINED anari_DIR)
+        set(anari_DIR ${VISIT_ANARI_DIR}/lib/cmake/anari-${ANARI_VERSION}
+            CACHE PATH
+            "The directory containing the ANARI config files."
+            FORCE)
+    endif(NOT DEFINED anari_DIR)
+
+    find_package(anari)
+endif()
+
+if(anari_FOUND)
+    add_definitions(-DVISIT_ANARI)
+
+    # Include directories
+    get_target_property(_INCLUDE_DIRS anari::anari INTERFACE_INCLUDE_DIRECTORIES)
+    set(ANARI_INCLUDE_DIRS ${_INCLUDE_DIRS} CACHE PATH "ANARI include directories" FORCE)
```

Comment:
Will the next PR directly utilize the Anari CMake targets?
Or will it use the combination of `ANARI_INCLUDE_DIRS` and `ANARI_LIBRARIES`?
If the Anari CMake targets work within VisIt, then that is the preferred method (and `ANARI_INCLUDE_DIRS` and `ANARI_LIBRARIES` cache vars wouldn't necessarily be needed).